src/outpack/util.py

```diff
@@ -82,6 +82,28 @@ def assert_file_exists(path, *, workdir=None, name="File"):
         raise Exception(msg)
 
 
+def assert_relative_path(path: str, name: str):
+    path = Path(path)
+
+    # On Windows, this is not equivalent to path.is_absolute().
+    # There are paths which are neither absolute nor relative, but somewhere
+    # between the two. These are paths such as `C:foo` or `\foo`. The former has
+    # a drive but no root, and the latter has a root but no drive. We want to
+    # exclude both scenarios. On POSIX, drive will always be empty and this is
+    # equivalent to calling `is_absolute()`.
+    #
+    # See https://github.com/python/cpython/issues/44626 for some discussion.
+    # Unfortunately, while the issue was closed, the `is_relative` function
+    # mentioned was never added.
```

Comment:
Looks like in 3.13+ we can use this! https://github.com/python/cpython/pull/113829 but for now this seems sensible